test shouldEnableStreamingWithProperty fails when building locally

**Summary**

> Try to answer the following as best as possible

- How often does the test fail?
Seems like it fails all the time when I try to build zeebe locally using 'mvn clean package' in the zeebe subdirectory
- Does it block your work?
No, I just want to make sure I can build the entire mono-repo locally if necessary
- Do we suspect that it is a real failure?

**Failures**

> Outline known failure cases, e.g. a failed assertion and its stacktrace obtained from Jenkins

<details><summary>Example assertion failure</summary>
<pre>
[ERROR] Tests run: 57, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.267 s <<< FAILURE! -- in io.camunda.zeebe.client.ZeebeClientTest
[ERROR] io.camunda.zeebe.client.ZeebeClientTest.shouldEnableStreamingWithProperty -- Time elapsed: 0.018 s <<< FAILURE!
org.opentest4j.AssertionFailedError: 

Expecting value to be true but was false
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at io.camunda.zeebe.client.ZeebeClientTest.shouldEnableStreamingWithProperty(ZeebeClientTest.java:186)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
</pre>
</details>

**Hypotheses**

> List any hypotheses if you have one; can be ommitted

**Logs**

> If possible, provide more context here, e.g. standard output logs, link to build, etc.

<details><summary>Logs</summary>
<pre>
LOGS
</pre>
</details>

--- a/zeebe/clients/java/src/test/java/io/camunda/zeebe/client/impl/worker/JobWorkerImplTest.java
+++ b/zeebe/clients/java/src/test/java/io/camunda/zeebe/client/impl/worker/JobWorkerImplTest.java
@@ -26,6 +26,7 @@ import io.camunda.zeebe.client.api.worker.JobWorkerBuilderStep1.JobWorkerBuilder
 import io.camunda.zeebe.client.impl.ZeebeClientBuilderImpl;
 import io.camunda.zeebe.client.impl.ZeebeClientImpl;
 import io.camunda.zeebe.client.impl.util.Environment;
+import io.camunda.zeebe.client.impl.util.EnvironmentRule;
 import io.camunda.zeebe.client.impl.util.ExecutorResource;
 import io.camunda.zeebe.gateway.protocol.GatewayGrpc;
 import io.camunda.zeebe.gateway.protocol.GatewayGrpc.GatewayImplBase;
@@ -68,6 +69,7 @@ public final class JobWorkerImplTest {
   private static final Duration SLOW_POLL_THRESHOLD = Duration.ofMillis(SLOW_POLL_DELAY_IN_MS / 2);
 
   @Rule public final GrpcCleanupRule grpcCleanup = new GrpcCleanupRule();
+  @Rule public final EnvironmentRule environmentRule = new EnvironmentRule();
 
   private MockedGateway gateway;
   private ZeebeClient client;

